internal/controller/api/management.go

```diff
@@ -25,8 +31,10 @@ func NewManagementServer(cm *controller.ConnectionManager, r *mux.Router) *Manag
 }
 
 func (s *ManagementServer) Routes() {
-	s.router.HandleFunc("/connection/disconnect", s.handleDisconnect())
-	s.router.HandleFunc("/connection/status", s.handleConnectionStatus())
+	securedSubRouter := s.router.PathPrefix("/connection").Subrouter()
```

Comment:
Should we make the prefix here /api/connection?

Reply:
Eventually, I want to make the prefix configurable and follow the standards from the interface IPP doc.